<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAssetDialog</class>
 <widget class="QDialog" name="AddAssetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QComboBox" name="assetList">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="addButton">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Assign asset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLineEdit" name="shortNameEdit">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>NEW</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="placeholderText">
          <string>New Asset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="createButton">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Create new asset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAssetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shortNameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>nameEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
